tests/libtest/test_ibm_hci_nodes.py

```
@@ -295,5 +296,63 @@ def test_node_operations_with_ceph_health_check(self):
+        ibm_hci_obj = IBMHCI()
+
+        # Derive domain suffix from IBMHCINode (reuses existing helper)
+        ibm_hci_node = IBMHCINode()
+        domain_suffix = ibm_hci_node.domain_suffix
+
+        failed_nodes = []
+
+        for rack_serial, rack_data in ibm_hci_obj.rack_details.items():
+            rack_ip = rack_data.get("rackInfo", {}).get("rackIP")
+            if not rack_ip:
+                logger.warning(f"Skipping rack {rack_serial}: no rackIP available")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)